File: lab06.py
#User Input
a = int(input('A value for a: '));
b = int(input('A value for b: '));
gamma = int(input('Input an angle in degrees: '));

#Importing the math module
import math
#Converting degrees to radians
radians = math.radians(gamma)

#The equation for the law of cosines. 
c = math.sqrt((a**2)+(b**2)-2*a*b*math.cos(radians)); 

print('The distance c is equal to: ', c);

File: lab06.m


A = double(input('Input a radias for r(A): ')); 
thetaA = double(input('Input your angle in degreesfor A: ')); 
B = double(input('Input a radias for r(B): ')); 
thetaB = double(input('Input your angle in degrees for B: ')); 

% x = A*cos(thetaA); 
% y = B*cos(thetaB); 

theta1 = (pi/180)*thetaA;
theta2 = (pi/180)*thetaB; 

c = ((A^2)+(B^2)-(2*A*B)*cos(theta1-theta2))^(1/2); 

disp(c)